transports/daily/package.json

```diff
@@ -30,12 +30,13 @@
     "@pipecat-ai/client-js": "~0.3.2"
   },
   "dependencies": {
-    "@daily-co/daily-js": "^0.73.0"
+    "@daily-co/daily-js": "^0.73.0",
+    "@pipecat-ai/realtime-websocket-transport": "~0.3.4"
```

Comment:
i know this might be crazy... but at this point it might make sense to pull the library of raw media management stuff into it's own library that these transports all depend on, rather than making the daily transport rely on the realtime-websocket-transport (seems weird)

Reply:
Yeah, I agree, it does feels a little bit weird.
Should we keep this library in the same repo ?